doc/source/user/teardown-socok8s.rst

```diff
@@ -1,25 +1,25 @@
-===============================
-Deleting socok8s from OpenStack
-===============================
+====================================================
+Deleting SUSE Containerized OpenStack from OpenStack
+====================================================
 
-If you have built socok8s on top of OpenStack, you can delete your whole
-environment by running:
+If you have built SUSE Containerized OpenStack (SCO) on top of OpenStack, you can
```

Comment:
This si only related to the AIO openstack deployment, so it wont work if you bring your own caasp/ses so it should be noted

Reply:
recommend addressing after merge